microwave/rollingpin fix

ITCups opened the pull request:
Микроволновка принимает стак предметов с размер в 1 единицу(раньше принимала только если в стаке больше 1)
Скалка теперь звучит не как режущее

Commit:
jmerge scripts for exodus map

Commit:
microwave now accept stacks of size 1

Commit:
rolling pin making bash sound now

Epicus7 commented on code/game/machinery/kitchen/microwave.dm:
```diff
@@ -48,6 +48,8 @@
 		acceptable_items |= /obj/item/weapon/holder
 		acceptable_items |= /obj/item/weapon/reagent_containers/food/snacks/grown
 
+
```
Эти изменения нужно убрать

Epicus7 commented on code/game/machinery/kitchen/microwave.dm:
```diff
@@ -109,9 +111,11 @@
 				new O.type (src)
 				S.use(1)
-				user.visible_message( \
+			else
```
>Микроволновка принимает стак предметов с размер в 1 единицу

Это не так работает.

Насколько я вижу, сейчас у тебя если больше единицы, то добавляется в микроволновку. А если один или ноль (так может быть?), то предмет выкинется на пол?

ITCups replied:
"выкидывает" в микроволновку.
`user.drop_item(src)` где (src) это указатель на микроволновку
Я тестил и без проверки не коммичу
И ноль не может быть так как он бы не прошел проверку выше

ITCups replied:
по сути `S.use(1)` это использовать один предмет из стака, но я не знал будут ли проблемы со стаком если я доюзаю его до нуля. Решил не рисковать и запихнул все оставшиеся в микроволновку. Для не стаковых предметов везде используется `user.drop_item(src)`

Epicus7 commented:
Еще надо убрать скрипты, опять же.

Commit:
Revert "jmerge scripts for exodus map"

This reverts commit 87905782445a51c3b412937e9f8c6df36c192030.

Commit:
removed empty line

Commit:
removed another empty line

Epicus7 committed:
Misc uniforms fixed

ITCups committed:
jmerge scripts for exodus map 

pull request #159

FighterX2500 committed:
Больше строк для фортепиано

pull request #161

ITCups committed:
Greytide virus fix

Грейтайд вирус теперь открывает входные шлюзы в бриге

fix #32

ITCups committed:
Blackscreen begone

pull request #152

ITCups committed:
"centcom_shuttle_dock_airlock duplicate program" error fix 

pull request #160

Epicus7 committed:
Hotfix of "Greytide virus fix" commit

ITCups committed:
New Robotistics, minor map fixes (#138)

* new robotistics
* added shutters to xenobiology and rd office
* added fire alarm in toxins
* moved fire alarm in xenobiology
* added rack to hold screewdriver adn wrench in toxins
* added blood to closet for medbay
* added surgery kits
* mech fabricator spawn items to the right of itself
* oxygen pump's mask fixed

Commit:
Exodus-6 Shuttle Fix, Research Outpost Cameras

- Починил инженерный шаттл, теперь шлюзы открываются так, как должны.
- Вернул камеры на научный аванпост.

Commit:
Exodus-2: new Robotics, Atmos and Bar + minor fixes

// Dnivji
- Добавил новый атмос вместе с новым нижнем уровнем для него.

// IT_Cups
- Добавил новый бар.
- Добавил анализаторы роботов в фабрикатор.

//hekko4
- Починил и склеил новые куски карты.
- Фиксики труб в токсинах.
- Убрал огромные стационарные скрабберы из РнД и пермабрига по причине своей неработоспособности.

Epicus7 committed:
Species preparation for take off
- Added species jobs restrictions
- Removed unathi skill
- Added temporary ingame whitelist

Commit:
Exodus-1 Fixes and new UndergroundAtmos room
- Утечки и невидимые трубы были исправлены.
- Большинство помещений имеет теперь свою собственную зону.
- Новая комнатка атмосферного отсека. [By Dnivji]. // Подправлена, а так же соединена с новым атмосом.

ITCups committed:
Bot fixes (#173)

- Боты теперь снова могут патрулировать станцию
- Мулботы теперь работают
- Боты больше не бессмертны
- С возвращением, Бипски

Epicus7 committed:
Changelog updated

Commit:
Merge branch 'kebab-fix' of https://github.com/LagannPM/Chaotic-Onyx into kebab-fix
